workflows/diagnostics/fv3net/diagnostics/prognostic_run/load_run_data.py

```diff
@@ -250,7 +202,22 @@ class SegmentedRun:
 
     @property
     def data_2d(self) -> xr.Dataset:
-        return load_2d(self.url, self.catalog)
+        url = self.url
+        catalog = self.catalog
+        path = os.path.join(url, "atmos_dt_atmos.zarr")
+        diags_url = os.path.join(url, "diags.zarr")
+        sfc_dt_atmos_url = os.path.join(url, "sfc_dt_atmos.zarr")
+
+        return xr.merge(
+            [
+                load_coarse_data(path, catalog),
+                # TODO fillna required because diags.zarr may be saved with an
```

Comment:
Is `fillna` also appropriate (if not necesary) for the fortran data? If so, I could push it into `load_coarse_data` and cleanup/generalize this logic.

Reply:
I think we could get rid of the `fillna`... it was there because of an issue with the `ZarrMonitor` that was fixed a long time ago IIRC.